pages/login-page/login-page.css

```diff
@@ -0,0 +1,60 @@
+@import url('/main.css');
+@import url('https://fonts.googleapis.com/css2?family=Prompt:ital@1&display=swap');
+
+.grid-wrapper{
+    display: grid;
+    grid-template-rows: auto 1fr auto;
+}
+.navbar{
+    grid-row: 1/2;
+}
+.signin-container{
+    grid-row: 2/3;
+    height: 85vh;
+}
+.form-wrapper{
+    margin: 2rem;
+    padding: 2rem;
+    display: flex;
+    flex-direction: column;
+    max-width: 20rem;
+    margin-left: auto;
+    margin-right: auto;
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+}
+.checkbox-div{
+    padding: 0.5rem;
+    display: flex;
+    justify-content: flex-start;
+    gap: 0.5rem;
+}
+.input{
+    padding: 0.5rem;
+    caret-color: var(--color-primary);
+    border: 2px solid var(--color-primary);
+    font-size: 1rem;
+    width: 100%;
+}
+.btn-login{
+    width: 100%;
+    margin: 1rem 0rem 0rem 0rem;
+    border-radius: 0rem;
+}
+footer{
+    grid-row: 3/4;
+    background-color: var(--color-primary);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color: var(--color-white)
+}
+.footer-list-container{
```

Comment:
Try to use only two names in naming classes

Reply:
Sure, will take care.